### ISSUE TITLE
Correctly parse attributes starting with colon (:)

### ISSUE THREAD

**Issue by qm3ster** [pugjs/pug#2771]
```pug
img(v-if="url" :src = "src" :srcset = "srcset" :alt = "alt")
//- -----------^ Unexpected token
img(v-if="url", :src = "src", :srcset = "srcset", :alt = "alt")
//- Great success!
```
This is distinct from [Quoted Attributes](https://pugjs.org/language/attributes.html#quoted-attributes) because here the addition of a comma completely saves the cheerleader. 
Furthermore, disaster only strikes an attribute that isn't first in the tag.  

```pug
img(:src = "src" :srcset = "srcset" :alt = "alt" v-if="url")
//- -------------^ Unexpected token
```
Lastly, code highlighters, including the ones used on GitHub and in Atom, think that newline is a solution, while the actual parser disagrees. Do highlighters not use the normal parser?
```pug
img(:src = "src" 
    :srcset = "srcset"
    :alt = "alt" v-if="url")
//- ^ Unexpected token (second line, :srcset)
```

**Comment by bitencode** [comment]
@qm3ster See #2454

**Comment by qm3ster** [comment]
Oh dang. Search really isn't what it used to be.
Closed as duplicate in favor of #2454
